test/Address.spec.js

```diff
@@ -12,7 +12,7 @@ describe('Address Resource', function() {
       shippo.address.retrieve('addressIdFoo123');
       expect(shippo.LAST_REQUEST).to.deep.equal({
         method: 'GET',
-        url: '/v1/addresses/addressIdFoo123',
```

Comment:
for all the tests that use URLs, let's import the `Shippo.DEFAULT_BASE_PATH` and interpolate it when forming the URL so that we don't have to make a change to all tests everytime the base path changes.